src/Compilers/Core/Portable/Serialization/ObjectWriter.cs

```diff
@@ -1307,8 +1307,8 @@ internal enum EncodingKind : byte
             EncodingName,
 
             // well-known encodings (parameterized by BOM)
-            EncodingUTF8,
-            EncodingUTF8_BOM,
+            EncodingUtf8,
+            EncodingUtf8_BOM,
```

Comment:
It looks like these should be kept as-is, or "EncodingUTF32*" should be updated as well.